akka-persistence-tck/src/main/scala/akka/persistence/journal/JournalSpec.scala

```diff
@@ -181,8 +183,11 @@ abstract class JournalSpec(config: Config) extends PluginSpec(config) {
       journal ! ReplayMessages(0, Long.MaxValue, Long.MaxValue, pid, receiverProbe.ref)
       receiverProbe.expectMsg(RecoverySuccess(highestSequenceNr = 5L))
     }
+  }
 
-    "reject non-serializable events" in {
+  "A Journal" may {
+    "reject non-serializable events" in optional("This journal does not reject non-serializable events. This is legal.") {
```

Comment:
Then I don't think the test is worth much, since the plugin maintainer will not see test failures no matter how things are implemented (or broken by regression), apart from parsing logs.

I think it's better to have a method that the plugin can override to say that it doesn't support rejections. I think we already have something like that for atomic `persistAll`.